src/BizHawk.Client.EmuHawk/MainForm.Events.cs

```diff
@@ -690,47 +690,36 @@ private void ViewSubMenu_DropDownOpened(object sender, EventArgs e)
 			DisplayMessagesMenuItem.Checked = Config.DisplayMessages;
 		}
 
-		private void WindowSizeSubMenu_DropDownOpened(object sender, EventArgs e)
+		private IReadOnlyCollection<ToolStripItem> CreateWindowSizeFactorSubmenus(int max_factor)
 		{
-			x1MenuItem.Checked =
-				x2MenuItem.Checked =
-				x3MenuItem.Checked =
-				x4MenuItem.Checked =
-				x5MenuItem.Checked = 
-				mzMenuItem.Checked = false;
+			List<ToolStripItem> items = new();
+			for (int i = 1; i <= max_factor; i++)
+			{
+				var temp = new ToolStripMenuItemEx()
+				{
+					Tag = i,
+					Text = $"{i / 10}&{i % 10}x"

```

Comment:
Clever. I'd like to keep "1x" rather than "01x" though. (There should be a `DivMod` somewhere too.)